src/server/scriptInfo.ts

```diff
@@ -176,8 +176,13 @@ namespace ts.server {
                 return this.switchToScriptVersionCache();
             }
 
-            // Else if the svc is uptodate with the text, we are good
-            return !this.pendingReloadFromDisk && this.svc;
+            // If there is pending reload from the disk then, reload the text
+            if (this.pendingReloadFromDisk) {

```

Comment:
Should `this.pendingReloadFromDisk` be set to false here or in a callee afte the reload completes?

Reply:
It does get set whenever the text is reloaded into the script info. (https://github.com/Microsoft/TypeScript/pull/22151/files#diff-f56eebffea5b0d2286bb5998c67920e9L78)

Reply:
Missed that. Thanks!